stable/astro/templates/configmap-data.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.custom_config.enabled }}
+{{- if and .Values.custom_config.enabled (not .Values.custom_config.configMapOverride) }}
```

Comment:
If the maintainers of astro don't like this then you might also use `include` function to allow consumers of this chart to pass partials containing the `config.yml` values, while still allowing the chart to maintain control over the `ConfigMap` resource.

Reply:
Max and I chatted about this. For the moment, we're in agreement that passing partials doesn't seem worth attempting because difficulties composing parent and child charts in Helm.